app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimDatePage.scala.html

```diff
@@ -80,9 +80,9 @@
     }
 }
 
+@defining(LocalDate.now().getYear - 1) { dynamicYear =>
```

Comment:
I don't think you need the `@defining` on lines 83 and 84. You just need to define `dynamicHintDate` in a similar way to how `@title` is defined. Happy to show you what I mean if its not clear/you're not sure.